layouts/default.vue

```diff
@@ -1,8 +1,7 @@
 <template>
   <div>
     <HeaderBlock />
-    <head-navigation />
-    <nuxt />
+    <nuxt class="py-8" />
```

Comment:
python? :D Just kidding but honestly I'm curious what is py here?

Reply:
py-8 = padding on y-axis with 8 units (8*0.25rem)

Reply:
So there's also my and px and mx? Thanks!